spec/Drivetrain/BatteryManagement.vspec

```diff
@@ -10,25 +10,47 @@
 # Battery management system for EV / Hybrids
 #
 
-- BatteryTemperature:
+- Battery:
```

Comment:
I have a proposal to do several things:

1. Move BatteryManagement from Drivetrain to Powertrain branch. Due to Wikipedia Drivetrain is the wrong location for this leafs.  The powertrain term includes all of the components used to transform stored (chemical, solar, nuclear, kinetic, potential, etc.) energy into kinetic energy for propulsion purposes. Drivetrain is a group of components that deliver power to the driving wheels.

2. In a car we have different Batteries.
    a. Automotive battery -->  Its main purpose is to feed the starter, which starts the engine.
    b. Electric vehicle battery --> Power the electric motor.

So I think we need also differentiation also hear. I am not native speaker in english but I would like to propose following:

- Vehicle.Powertrain.Storage.HighVoltage --> for Electric vehicle battery
- Vehicle.Powertrain.Storage.AutomobileBattery --> automobile battery

or

- Vehicle.Powertrain.Battery.ElectricVehicle --> for Electric vehicle battery
- Vehicle.Powertrain.Battery.Automobile --> automobile battery

Reply:
Very good point, Alexander.  I think this PR might have by mistake removed the normal car battery because everyone is so focused on EVs?

> In a car we have different Batteries.
> a. Automotive battery --> Its main purpose is to feed the starter, which starts the engine.

In fact, often also called the Starter Battery I think?  I'm not sure why the other battery would not be "automotive"?

> or
> Vehicle.Powertrain.Battery.ElectricVehicle --> for Electric vehicle battery
> Vehicle.Powertrain.Battery.Automobile --> automobile battery

or even the opposite, in my opinion.  The branch named ElectricVehicle should be higher up since the whole branch can be avoided completely if it is not an EV:

Vehicle.Powertrain.Battery.**ElectricVehicle** --> Vehicle.Powertrain.**ElectricVehicle**.Battery  ?

Reply:
EV Battery under powertrain -> see comment @gunnarx.
Support battery in a different issue, as not only EV.

Reply:
vehicle.powertrain.electric.battery

Reply:
Done as discussed.